gdal/frmts/ceos2/sar_ceosdataset.cpp

```diff
@@ -1558,6 +1643,17 @@ void SAR_CEOSDataset::ScanForGCPs()
         return;
     }
 
+    /* ASF L1 products do not have valid data
+       in the lat/long first/mid/last fields */
+    const char *pszValue = GetMetadataItem("CEOS_FACILITY");
+    if(pszValue == nullptr)
+        return;
```

Comment:
I don't know anything about that format, but isn't this return too radical ? Shouldn't just the non-null check be adding to the following strncmp() ?

Reply:
apologies, fixed